src/components/a-input/_template.js

```diff
@@ -20,9 +20,9 @@ export default function ({
           value="${value}"
           disabled="${disabled}">
           <span class="a-input__valid-icon"></span>
-          ${icon ? html`<button class="a-input__icon__button js-input__icon__button" type="button">
+          ${icon ? html`<button class="a-input__icon-button js-input__icon-button" type="button">
```

Comment:
@LucaMele 
@jackblackCH 
hmh, may off-topic.
But why do we have a `<button>` next to `<input />` ?

Reply:
@AndyOGo cause a input has a "icon" button for an action on the website

Reply:
**Off-Topic**:
Interesting.
But why is it combined with basic `<axa-input>` 🤔 
Shouldn't it be separated.

Reply:
no i think is ok, as is a input having a button make sense that the button is inside the input ;)

Reply:
hmh, okay.
I would rather solve it with composition, i.e. have a third component which composes `<axa-input>` and `<button>` to keep concerns separated.

Reply:
could also be ok